examples/express-all-interactions/.env.sample

```diff
@@ -2,5 +2,5 @@
 SLACK_VERIFICATION_TOKEN=
 SLACK_ACCESS_TOKEN=
 
-# Optional
-PORT=
+# Uncomment PORT and fill out if local (not on Glitch)
```

Comment:
I don't think this is as clear as it could be. Perhaps `# Optional - not necessary on Glitch`